lib/rubocop/cop/style/inverse_methods.rb

```diff
@@ -46,6 +46,7 @@ class InverseMethods < Base
 
         MSG = 'Use `%<inverse>s` instead of inverting `%<method>s`.'
         CLASS_COMPARISON_METHODS = %i[<= >= < >].freeze
+        INCOMPATIBLE_INVERSE_METHODS = %i[any? none?].freeze
```

Comment:
I'd probably add a comment here mentioning they are incompatible only in the context of safe navigation.